models/dex/dex_info.sql

```diff
@@ -146,11 +146,11 @@ FROM (VALUES
     , ('syncswap', 'SyncSwap', 'Direct', 'SyncSwap')
     , ('mute', 'Mute', 'Direct', 'Mute')
     , ('spacefi', 'SpaceFi', 'Direct', 'SpaceFi')
-    , ('derpdex', 'DerpDEX', 'Direct', 'DerpDEX')
```

Comment:
any particular reason this dex is being removed?

Reply:
it is duped with entry in line 124